### PR TITLE
Guard rework

### PR DESCRIPTION
**Pull request by emoats18**
Reworks the guards to make them more akin to a professional garrison.

Changes veteran into the Captain of the Guard.

Disables sheriff in favor for the captain of the guard, Bailiff basically replaced him anyway.

Gives guards better equipment and makes their loadouts make more sense.

**CASTLE GUARD**
castleguard
castleguard1

**TOWN GUARD**
townguard
townguard1

**VETERAN (RENAMED TO CAPTAIN OF THE GUARD)**
veteran
captainoftheguard

### CODE REVIEW

**Comment by kacperpl1234pl** [comment]
making guards better geared with adventurer pop nerf gona ensure 95% of rebelions will fail

**Comment by CVSthePharmacy** [comment]
>gives gaurds better equipment

bro you just nerfed castle guards by giving them the shittier helmet and a CHAIN COIF instead of their GORGET, have you SEEN the different in the HP of those two?

**Comment by TheManWithNoHands** [comment]
Mate, give the keep's armory some more steel equipment as well then. I mainly put iron gear in there to not give the guards access to stronger gear than what they spawn with. It's under the keep's guard quarters.


> making guards better geared with adventurer pop nerf gona ensure 95% of rebelions will fail

Good, having rebellion 95% of the time succeed whenever they happen is very unfun. In general Castle guards are kinda weak rigth now. So it will be healthy if they actually get proper gear.

**Comment by CVSthePharmacy** [comment]
> Mate, give the keep's armory some more steel equipment as well then. I mainly put iron gear in there to not give the guards access to stronger gear than what they spawn with. It's under the keep's guard quarters.
> 
> > making guards better geared with adventurer pop nerf gona ensure 95% of rebelions will fail
> 
> Good, having rebellion 95% of the time succeed whenever they happen is very unfun. In general Castle guards are kinda weak rigth now. So it will be healthy if they actually get proper gear. Always feels like their just an honor guard.

I've still gotta do my rebellion speedrun when rolling knight and getting rebel leader

**Comment by TheManWithNoHands** [comment]
> I've still gotta do my rebellion speedrun when rolling knight and getting rebel leader

No way knigth is able to rebel leader. You lying.

**Comment by TheManWithNoHands** [comment]
> I rolled it a few weeks ago

Dayum

**Comment by Real-MAGNUM** [comment]
> > > I've still gotta do my rebellion speedrun when rolling knight and getting rebel leader
> > 
> > 
> > No way knigth is able to rebel leader. You lying.
> 
> I rolled it a few weeks ago

They were disabled since then iirc.

**Comment by Real-MAGNUM** [comment]
Visual distinction between castle and town is lacking
Return the helmet and gorget on the castle guard and it's a fine PR

**Comment by CVSthePharmacy** [comment]
> > > > I've still gotta do my rebellion speedrun when rolling knight and getting rebel leader
> > > 
> > > 
> > > No way knigth is able to rebel leader. You lying.
> > 
> > 
> > I rolled it a few weeks ago
> 
> They were disabled since then iirc.

can you re-add them

**Comment by CVSthePharmacy** [comment]
also can you give the vet a STEEL mace or a GRAND mace instead of a normal lame mace that a WOODEN STAFF can outperform?

File: code/modules/jobs/job_types/roguetown/garrison/bogmaster.dm [hunk 1]
```diff
@@ -43,31 +44,33 @@
...
+	wrists = /obj/item/clothing/wrists/roguetown/bracers
 	beltr = /obj/item/rogueweapon/sword
 	belt = /obj/item/storage/belt/rogue/leather
 	backr = /obj/item/storage/backpack/rogue/satchel
 	backl = /obj/item/rogueweapon/shield/tower
 	backpack_contents = list(/obj/item/rogueweapon/huntingknife/idagger/steel/special = 1)
 	if(H.mind)
 		H.mind.AddSpell(new /obj/effect/proc_holder/spell/self/convertrole/bog)
-		H.mind.adjust_skillrank(/datum/skill/combat/axesmaces, 3, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/combat/axesmaces, 6, TRUE)
```
Comment (Real-MAGNUM) [inline code/modules/jobs/job_types/roguetown/garrison/bogmaster.dm:62]:
```suggestion
		H.mind.adjust_skillrank(/datum/skill/combat/axesmaces, 4, TRUE)
```

File: code/modules/jobs/job_types/roguetown/garrison/veteran.dm [hunk 2]
```diff
@@ -15,8 +15,8 @@
 		"Aasimar",
 		"Half Orc",
 	) //same as town guard
-	tutorial = "You've served the garrison your whole life. There isn't a way to kill a man you havent practiced in the tapestries of war itself. You wouldn't call yourself a hero, those belong to the men left rotting in the fields of where you practiced your ancient trade. You don't sleep well at night anymore, you don't like remembering what you've had to do to survive. Trading adventure for stable pay was the only logical solution, and maybe someday you'll get to lay down the blade..."
-	allowed_ages = list(AGE_OLD)
+	tutorial = "War has always been a constant of your life, and you always chose the side of defense and justice. You rose up through the ranks as a guardman and you're now the head above all of them. Lead and train your men to defend the kingdom and maintain the peace. Recruit those who feels able to aid the Kingdom against the offensives and push the tides to be yours..."
```
Comment (Real-MAGNUM) [inline code/modules/jobs/job_types/roguetown/garrison/veteran.dm:18]:
```suggestion
	tutorial = "War has always been a constant of your life, and you always chose the side of defense and justice. You rose up through the ranks as a guardman and you're now the head above all of them. Lead and train your men to defend the kingdom and maintain the peace. Recruit those who you believe capable of aiding the Kingdom against offenders and pushing the tides of the kingdom's battles in your favor..."
```

**Review by Real-MAGNUM — CHANGES_REQUESTED** [review]
Could use some balance tweaking.

File: code/modules/jobs/job_types/roguetown/garrison/veteran.dm [hunk 3]
```diff
@@ -25,19 +25,34 @@
...
-	neck = /obj/item/clothing/neck/roguetown/gorget
-	shoes = /obj/item/clothing/shoes/roguetown/boots
+	shirt = /obj/item/clothing/suit/roguetown/armor/chainmail
+	armor = /obj/item/clothing/suit/roguetown/armor/plate
+	neck = /obj/item/clothing/neck/roguetown/bervor
+	shoes = /obj/item/clothing/shoes/roguetown/boots/armor
 	beltl = /obj/item/keyring/guardcastle
-	belt = /obj/item/storage/belt/rogue/leather
-	backr = /obj/item/storage/backpack/rogue/satchel
+	beltr = /obj/item/rogueweapon/mace
```
Comment (Real-MAGNUM) [inline code/modules/jobs/job_types/roguetown/garrison/veteran.dm:49]:
```suggestion
	beltr = /obj/item/rogueweapon/mace/steel
```

File: code/modules/jobs/job_types/roguetown/garrison/bogmaster.dm [hunk 4]
```diff
@@ -43,31 +44,33 @@
...
 		H.mind.adjust_skillrank(/datum/skill/combat/bows, 4, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/combat/crossbows, 5, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/combat/wrestling, 4, TRUE)
 		H.mind.adjust_skillrank(/datum/skill/combat/unarmed, 3, TRUE)
-		H.mind.adjust_skillrank(/datum/skill/combat/swords, 4, TRUE)
+		H.mind.adjust_skillrank(/datum/skill/combat/swords, 6, TRUE)
```
[the remark this answers is not in the document]
Comment (CVSthePharmacy) [reply code/modules/jobs/job_types/roguetown/garrison/bogmaster.dm:67]: also give them 6, they are a training role

**Comment by BlueMemesauce** [comment]
So why do the captain of the guard and sheriff both exist?

**Comment by Real-MAGNUM** [comment]
> So why do the captain of the guard and sheriff both exist?

I think he planned to replace it with the captain of the guard. It's the same role afaik.

**Comment by Wintermote** [comment]
Not a fan of turning the veteran into a captain of the guard. Should be 2 separate roles, captain of the guard is basically what the sheriff is, veteran seems more like an old guard who's been around the block, or a hired on advisor rather than the guy leading the guard.

Also the full plate near knight armor ruins the vet aesthetics, you can't even tell he has an eye patch anymore.

**Comment by Real-MAGNUM** [comment]
emoney testmerged and didnt tag it

**Comment by BlueMemesauce** [comment]
why dont you just get the bot that checks when prs are testmerged

**Comment by YosemiteYam** [comment]
Bad PR, why are we replacing the sheriff with the bailiff? They are NOT the same thing, the bailiff is a dumb rolebloat snowflake role you introduced and no one wanted, and ingame most bailiffs don't do shit but act as a fifth wheel to all the other advisors. You WANTED them to replace the sheriff but they have failed to do so repeatedly, with many complaints directed to how the councillors/bailiffs do basically nothing/contribute nothing regularly. Instead of town guard and bogmaster answering to the jacked sheriff that is actually a good military role you're expecting them to answer to the snowflake "judge" role that never actually judges anyone. 

Secondly you're getting rid of actual flavor in the court by turning the veteran role into the captain of the guard, basically removing veteran as a thing. 

This is a pretty big change disguised as a castle guard rework and should've been discussed more in depth before being test merged.

**Comment by BadAtThisGame302** [comment]
Now with the role Captain of the Guard, I don't think the name Sheriff fits if he is meant to be above the Captain, perhaps renaming that to like Marshal would be better? Not to mention that he can now spend time in the Court with the King advising him while the Captain keeps the Guard in check.

It can just add Court RP and more Councilmen to give advice to the King instead of everyone running off.

The Bailiff should be removed since now the Sheriff/Marshal can do his job while the Captain leads the Guard. 

Councillors can stick around and purely act as the Kings and his advisors councilmen, they just are minor nobles that have no authority except for commanding the Kings respect and their voices being heeded by him or the Hand/Steward/Sheriff etc.

File: code/modules/jobs/job_types/roguetown/garrison/veteran.dm (hunk 2 above)
Comment (BadAtThisGame302) [reply code/modules/jobs/job_types/roguetown/garrison/veteran.dm]: One Kingdom is capitalized while the other kingdom isn't. Should keep it the same. (Also perhaps remove the 2 extra dots.

**Comment by wow50000** [comment]
could you bring back the old helmets though they are cool as hell ?

**Comment by CVSthePharmacy** [comment]
bring back the old castle guard helmets they were cool and give them gorgets, other than that this is okayish only because of captain

**Comment by CVSthePharmacy** [comment]
> Not a fan of turning the veteran into a captain of the guard. Should be 2 separate roles, captain of the guard is basically what the sheriff is, veteran seems more like an old guard who's been around the block, or a hired on advisor rather than the guy leading the guard.
> 
> Also the full plate near knight armor ruins the vet aesthetics, you can't even tell he has an eye patch anymore.
> 
> In fact, this is really just removing veteran outright as a job, not changing it.

it's half plate, clerics get it, so shush

**Comment by Wirpy** [comment]
I like the equipment changes beyond the neck and helmet. 

I might be wrong but the helmet you've changed to in this pr is iron isn't it? Im fairly confident it is but I might be wrong. 

I'd also argue that a gorget is preferable to a chain coif but that might be personal preference.  Otherwise its nice and feels more professiona lto finally have chainmail underneath roundstart lol and stuff.

**Pull request closed without merging**